public/js/combinationalAnalysis.js

```diff
@@ -101,8 +101,24 @@ function createBooleanPrompt(inputListNames,outputListNames,scope=globalScope){
             // //console.log(dataSample);
             drawCombinationalAnalysis(minmizedCircuit,inputListNames,outputListNames,scope)
         },
-
-        }
+      },
+      {
+        text: "Download",
+        click: function() {
+           var sTable = document.getElementById('combinationalAnalysis').innerHTML;
```

Comment:
can we change this variable to booleanLogicTable? I think this way it's better to understand what this variable contains
Can we make this as constant? const booleanLogictable=....